pkg/operator/controller/controller_dns_daemonset_test.go

```diff
@@ -104,6 +113,13 @@ func TestDaemonsetConfigChanged(t *testing.T) {
 			},
 			expect: true,
 		},
+		{
+			description: "if .spec.template.spec.volumes changes",
+			mutate: func(daemonset *appsv1.DaemonSet) {
+				daemonset.Spec.Template.Spec.Volumes = []corev1.Volume{volume}
```

Comment:
Is it necessary to use a named value rather than a literal?  If you prefer not to use a literal for stylistic reasons, consider moving the declaration of `volume` here as this is the only place that uses it.